cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is there a way to stop this from being created when running Cypress or do we need to add it to our .gitignore?

Reply:
Setting the fixtures folder should do the trick